src/status_im/utils/config.cljs

```diff
@@ -33,7 +33,7 @@
 
 ;; CONFIG VALUES
 (def log-level
-  (-> (get-config :LOG_LEVEL "error")
+  (-> (get-config :LOG_LEVEL "debug")
```

Comment:
default log-level in the absence of env file should stay error IMO why debug ?

Reply:
yep sorry, that wasn't intended

Reply:
fixed